src/tools/ApiChecker/ScaffoldTest.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.CodeDom;
+using System.CodeDom.Compiler;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace ApiChecker
+{
+    [SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1118:ParameterMustNotSpanMultipleLines", Justification = "CodeDom is complicated.")]
+    internal static class ScaffoldTest
+    {
+        private static readonly TextInfo _textInfo = CultureInfo.InvariantCulture.TextInfo;
+
+        public static Task Run(ScaffoldTestOptions options)
```

Comment:
If there is not async action, shouldn't `ParseArguments` support a `void` method?